### PR TITLE
Revert back to using requests in the Python log client.

### PR DESCRIPTION
**Pull request by eranmes**
This reverts commit 3499f4ff6bd20627e6a11ee13aa79d5f4d96da43.

### CODE REVIEW

File: python/ct/client/scanner.py [hunk 1]
```diff
@@ -105,11 +105,11 @@ def _scan(entry_queue, log_url, range_description):
         entries = client.get_entries(range_start, range_end)
         scanned = range_start
         for entry in entries:
+            scanned += 1
```
Comment (phad) [inline python/ct/client/scanner.py]: I can't remember why the original commit moved this.  It looks like it's intended to fix an off-by-one issue.

Comment (phad) [inline python/ct/client/scanner.py]: So I think you should not modify scanner.py in this commit.

Comment (eranmes) [inline python/ct/client/scanner.py]: Done.

**Comment by eranmes** [comment]
PTAL.

**Pull request merged**